src/include/hip/rpp_hip_common.hpp

```diff
@@ -1967,6 +1968,23 @@ __device__ void rpp_hip_rng_8_gaussian_f32(d_float8 *rngVals_f8, RpptXorwowState
     rngVals_f8->f2[3] = rpp_hip_rng_2_gaussian_f32(xorwowState);
 }
 
+__device__ __forceinline__ float2 rpp_hip_rng_2_jitter_f32(RpptXorwowStateBoxMuller *xorwowState)
+{
+    float2 rngVals_f2;
+    rngVals_f2.x = rpp_hip_rng_xorwow_f32(xorwowState);
+    rngVals_f2.y = rpp_hip_rng_xorwow_f32(xorwowState);
+
+    return rngVals_f2;
+}
+
+__device__ void rpp_hip_rng_8_jitter_f32(d_float8 *rngVals_f8, RpptXorwowStateBoxMuller *xorwowState)
```

Comment:
Same comment again, did the templating work? If yes, why do we need these - rpp_hip_rng_8_jitter_f32() and rpp_hip_rng_2_jitter_f32()?

Reply:
yes templating worked
I am removing those two functions